model/sharedStimulus/factory/CommandFactory.php

```diff
@@ -39,4 +41,16 @@ public function makeCreateCommandByRequest(ServerRequestInterface $request): Cre
             $parsedBody['languageId'] ?? $parsedBody['languageUri'] ?? null
         );
     }
+
+    public function patchStimulusByRequest(ServerRequestInterface $request, User $user): UpdateCommand
```

Comment:
```suggestion
    public function makeUpdateCommandByRequest(ServerRequestInterface $request, User $user): UpdateCommand
```

Lets use the naming convention `make` for the class, `patch` seems you are executing a procedure instead.